pom.xml

```diff
@@ -472,28 +478,34 @@
 
     <profiles>
         <profile>
-            <id>default</id>
-            <activation>
-                <activeByDefault>true</activeByDefault>
-            </activation>
-            <modules>
-                <module>dl</module>
-                <module>spark-version</module>
-            </modules>
+            <id>2.0</id>
+            <properties>
+                <spark-version.project>2.0</spark-version.project>
+                <spark.version>2.0.0</spark.version>
+            </properties>
+        </profile>
+        <profile>
+            <id>scala_2.11</id>
+            <properties>
+                <scala.major.version>2.11</scala.major.version>
+                <scala.version>2.11.8</scala.version>
+                <scala.macros.version>2.1.0</scala.macros.version>
+            </properties>
+        </profile>
+        <profile>
+            <id>all-in-one</id>
+            <properties>
+                <spark-scope>compile</spark-scope>
+            </properties>
         </profile>
         <profile>
             <id>full-build</id>
```

Comment:
This is not only build native module. The native module will be added into the building module list.